setup.py

```diff
@@ -15,7 +15,7 @@
     "lst1_magic_stereo_reco = magicctapipe.scripts.lst1_magic.lst1_magic_stereo_reco:main",
     "lst1_magic_train_rfs = magicctapipe.scripts.lst1_magic.lst1_magic_train_rfs:main",
     "magic_calib_to_dl1 = magicctapipe.scripts.lst1_magic.magic_calib_to_dl1:main",
-    "merge_hdf_files = magicctapipe.scripts.lst1_magic.merge_hdf_files:main",
+    "merge_hdf_files = magicctapipe.scripts.lst1_magic.merge_hdf_files:main",  
```

Comment:
one more of unneccesary spaces marking the line as changed

Reply:
Done